osu.Game/Screens/Play/Player.cs

```diff
@@ -110,7 +110,7 @@ public abstract class Player : ScreenWithBeatmapBackground, ISamplePlaybackDisab
 
         protected HealthProcessor HealthProcessor { get; private set; }
 
-        protected DrawableRuleset DrawableRuleset { get; private set; }
+        public DrawableRuleset DrawableRuleset { get; set; }
```

Comment:
This does not need to be exposed as public (much less so with a public setter). Use `IApplicableToDrawableRuleset` in the mod instead.

Reply:
Makes sense and will fix, thanks!